.github/ISSUE_TEMPLATE/epic.yaml

```diff
@@ -0,0 +1,89 @@
+name: Epic (Internal Use Only 🔒)
```

Comment:
To me, this one seems not as useful, but that might just be because I typically open very specific epics for MVPs, which have different format. So I'll caveat my critique by saying: see what other folks think.

Reply:
We should 100% delete if it's not going to be used! @brmur, @scmacdon, or @beqqrry-aws: any thoughts?

Reply:
It is true that we have largely created Epics based on scenarios and ccs's, but I see a future which does have us using this template. However, maybe we don't need this in here cluttering things until we start utilizing Epics in a more general sense? We can pull it out of this PR and easily come back and put this in when required.

Reply:
Epic! Less is more, I agree. I'll wait until Brian approves (he often creates Epics).